packs/spells/shock-to-the-system.json

```diff
@@ -28,7 +28,7 @@
             }
         },
         "description": {
-            "value": "<p>A dense lightning cloud descends to fill the target's space, then fires a jolt of revitalizing lightning into the target. If the target is a corpse that died within the last round, the creature comes back to life with 0 Hit Points, and any effects and conditions it had when it died, with the exception of dying, and its wounded condition increases by 1. The creature's initiative is right before yours.</p>\n<p>Regardless of whether the creature came back to life or was already alive, it regains 8d8 Hit Points, and the bolt wakes it up if it was @UUID[Compendium.pf2e.conditionitems.Item.Unconscious]. For the duration of the spell, the target is supercharged. It becomes @UUID[Compendium.pf2e.conditionitems.Item.Quickened] and can use the extra action to Stand, Stride, Strike, or Fly (if it has a fly Speed). In addition, it can cast 5th-rank <em>thunderstrike</em> as an innate spell at will, using your spell DC.</p>\n<p>The cloud that covered the creature remains until the end of the target's next turn. Any creature in the cloud is @UUID[Compendium.pf2e.conditionitems.Item.Hidden], and anything outside the cloud is hidden to any creature inside the cloud.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> The healing increases by 2d8, and the thunderstrike rank increases by 1.</p>"
+            "value": "<p>A dense lightning cloud descends to fill the target's space, then fires a jolt of revitalizing lightning into the target. If the target is a corpse that died within the last round, the creature comes back to life with 0 Hit Points, and any effects and conditions it had when it died, with the exception of dying, and its wounded condition increases by 1. The creature's initiative is right before yours.</p>\n<p>Regardless of whether the creature came back to life or was already alive, it regains 8d8 Hit Points, and the bolt wakes it up if it was @UUID[Compendium.pf2e.conditionitems.Item.Unconscious]. For the duration of the spell, the target is supercharged. It becomes @UUID[Compendium.pf2e.conditionitems.Item.Quickened] and can use the extra action to Stand, Stride, Strike, or Fly (if it has a fly Speed). In addition, it can cast 5th-rank @UUID[Compendium.pf2e.spells-srd.zDJS8E66UI0himqV]{Thunderstrike} as an innate spell at will, using your spell DC.</p>\n<p>The cloud that covered the creature remains until the end of the target's next turn. Any creature in the cloud is @UUID[Compendium.pf2e.conditionitems.Item.Hidden], and anything outside the cloud is hidden to any creature inside the cloud.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> The healing increases by 2d8, and the thunderstrike rank increases by 1.</p>"
```

Comment:
```suggestion
            "value": "<p>A dense lightning cloud descends to fill the target's space, then fires a jolt of revitalizing lightning into the target. If the target is a corpse that died within the last round, the creature comes back to life with 0 Hit Points, and any effects and conditions it had when it died, with the exception of dying, and its wounded condition increases by 1. The creature's initiative is right before yours.</p>\n<p>Regardless of whether the creature came back to life or was already alive, it regains 8d8 Hit Points, and the bolt wakes it up if it was @UUID[Compendium.pf2e.conditionitems.Item.Unconscious]. For the duration of the spell, the target is supercharged. It becomes @UUID[Compendium.pf2e.conditionitems.Item.Quickened] and can use the extra action to Stand, Stride, Strike, or Fly (if it has a fly Speed). In addition, it can cast 5th-rank @UUID[Compendium.pf2e.spells-srd.Item.Thunderstrike] as an innate spell at will, using your spell DC.</p>\n<p>The cloud that covered the creature remains until the end of the target's next turn. Any creature in the cloud is @UUID[Compendium.pf2e.conditionitems.Item.Hidden], and anything outside the cloud is hidden to any creature inside the cloud.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> The healing increases by 2d8, and the thunderstrike rank increases by 1.</p>"
```